packages/account/package.json

```diff
@@ -50,6 +50,7 @@
     "@biconomy/bundler": "^3.1.2",
     "@biconomy/modules": "^3.1.2",
     "@biconomy/paymaster": "^3.1.2",
+    "ethers": "^6.9.2",
```

Comment:
Should use [@ethersproject/providers@5.7.2](https://bundlephobia.com/package/@ethersproject/providers@5.7.2) instead of [ethers](https://bundlephobia.com/package/ethers@6.10.0)